iothub/device/src/ClientPropertiesUpdateResponse.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.Client
+{
+    /// <summary>
+    /// The response of an <see cref="InternalClient.UpdateClientPropertiesAsync(ClientPropertyCollection, System.Threading.CancellationToken)"/> operation.
+    /// </summary>
+    public class ClientPropertiesUpdateResponse
+    {
+        /// <summary>
+        /// The request Id that is appended to the <see cref="InternalClient.UpdateClientPropertiesAsync(ClientPropertyCollection, System.Threading.CancellationToken)"/> operation.
+        /// </summary>
+        /// <remarks>
+        /// This request Id is relevant only for operations over MQTT, and can be used for debugging the operation from the service side.
```

Comment:
Do users really have a use for this field? As far as I know, the SDK picks the value when it sends the request, and I've never seen anything on the service side that exposes this value to the user. I pictured it would only be useful for Kusto logs that only we would have access to

Reply:
You're correct, this would only be useful if users wanted to track some requests via a support ticket, similar to how they use a tracking Id with an exception. I don't think this will add value for every customer (or even most), but since we are returning a response body anyway (for the version), I thought I'd add it in.

Reply:
I'd add that additional bit of context to your remarks, then. Without it, it sounds like I'd be able to track requests by Id through the portal or something